src/wazuh_modules/vulnerability_detector/cpe_helper.json

```diff
@@ -55,8 +108,8 @@
                 "version": []
             },
             "action": [
-                "replace_vendor_if_matches",
-                "replace_product_if_matches"
+                "replace_vendor",
+                "replace_product"
```

Comment:
As mentioned in the [documentation](https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/cpe-helper.html#translation-types), when there are several cases that you can match, you need to add the `_if_matches` to the action.
```suggestion
                "replace_vendor_if_matches",
                "replace_product_if_matches"
```